### ISSUE TITLE
10a support

### ISSUE THREAD

**Issue by jacobbloy** [GaryA/antweight-esc#1]
Hello did you ever get 10a support working? Because of the 4s lipo compatibility!

**Comment by GaryA** [comment]
No, I've not needed to and nobody else has asked for it. The necessary changes will come down to any differences in the hardware design, assuming the same micocontroller is used. If it is the same microcontroller then it could be as simple as changing any pin mapping and polarity - there might not be any changes if the design is similar enough! But without any hardware it is very difficult to make progress.
If you just want a 10A controller you may be better of just getting one from Botbitz: http://www.botbitz.com/index.php?route=product/product&path=59&product_id=56 (unless you particularly want to do your own thing...
